org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -626,4 +628,11 @@ public static synchronized ISourceDownloader getDefaultSourceDownloader() {
 		}
 		return pluginInstance.sourceDownloader;
 	}
+
+	public static ExecutorService getExecutorService() {
```

Comment:
Add `synchronized` to make it thread-safe?

Reply:
Ok, makes sense!